doc/nrf/ug_partition_manager.rst

```diff
@@ -35,31 +36,35 @@ You can define three types of partitions:
 
 Configuration
 =============
-Each sub-image is required to define its Partition Manager configuration in a file called :file:`pm.yml`.
+Each sub-image is required to define its Partition Manager configuration in a file called :file:``pm.yml``.
+This file must be stored in the same folder as the :file:``CMakeLists.txt`` of that sub-image.
 
 .. note::
-   The root application does not define its own :file:`pm.yml` because its partition size
-   and placement is implied by the size and placement of the sub image
-   partitions.
+   :file:``pm.yml`` is only used for sub images.
```

Comment:
Sorry to have ben unclear -- double backticks are needed if you are writing an inline code literal, like this:

```
I want to refer to a ``code_variable`` and have it rendered verbatim, in monospaced font.
```

But you still only need one backtick for the argument to an interpreted text role, like this:

```
I want to refer to a file named :file:`pm.yml`.
```

(See https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html)

Reply:
Sorry, I was too quick. Should be fixed now, thanks.